Sources/Amplitude/Public/Amplitude.h

```diff
@@ -30,7 +30,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 typedef NSString *_Nonnull (^AMPAdSupportBlock)(void);
 typedef NSDictionary *_Nullable (^AMPLocationInfoBlock)(void);
-
+typedef void (^AMPInitializeCompleteSupportBlock)(void);
```

Comment:
The name is too long. How about `AMPInitCompletionBlock`?

Reply:
Much better! Will update the name to this.